ospfd/ospf_vty.c

```diff
@@ -10648,28 +10659,42 @@ static void ospf_interface_clear(struct interface *ifp)
 
 DEFUN (clear_ip_ospf_interface,
        clear_ip_ospf_interface_cmd,
-       "clear ip ospf interface [IFNAME]",
+       "clear ip ospf [vrf <NAME>] interface [IFNAME]",
        CLEAR_STR
        IP_STR
        "OSPF information\n"
+       VRF_CMD_HELP_STR
        "Interface information\n"
        "Interface name\n")
 {
-	int idx_ifname = 4;
+	int idx_ifname = 6;
```

Comment:
you need to use `argv_find()` here since this command has optional parameters.

Reply:
this is what is done later.
to have consistency, I use idx_ifname = 0